sunpy/map/tests/test_map_factory.py

```diff
@@ -86,6 +86,10 @@ def test_patterns(eit_fits_directory):
     # Pathlib
     path = pathlib.Path(AIA_171_IMAGE)
     aiamap = sunpy.map.Map(path)
+
+    maps = sunpy.map.Map(list(path.glob('*.fits')))
```

Comment:
Why does this need to be wrapped a in list first?

Reply:
Yeah! got it will correct it ASAP

Reply:
I have got a small question. Which class does this object belong to? list?

Reply:
Which object?

Reply:
Have you had the time to correct this?